bundles/org.openhab.ui/web/src/assets/definitions/widgets/plan/index.js

```diff
@@ -31,16 +31,17 @@ export const OhPlanMarkerDefinition = () => new WidgetDefinition('oh-plan-marker
     pt('coords', 'Coordinates', 'The coordinates of this marker in the floor plan Coordinate Reference System; usually set by dragging the marker at design time').a(),
     pi('item', 'Item', 'The item whose state to display on this marker')
   ])
-  .paramGroup(pg('icon', 'Icon', 'You can customize the styles further with CSS attributes in the <code>iconStyle</code> parameter (in YAML only)'), [
+  .paramGroup(pg('icon', 'Icon', 'You can customize the styles further with CSS attributes in the <code>iconStyle</code> parameter (in YAML only) - see example below'), [
```

Comment:
You can't really add this in the property description, because when it's presented in a config sheet, there will be nothing to see "below".